readme.md

```diff
@@ -109,8 +109,6 @@ try {
 		command: 'wrong command',
 		exitCode: 2,
 		exitCodeName: 'ENOENT',
-		stdout: null,
-		stderr: null,
```

Comment:
Shouldn't these be an empty string instead, like the async version? https://github.com/sindresorhus/execa/pull/246/files#diff-0730bb7c2e8f9ea2438b52e419dd86c9R72

Reply:
Fixed: https://github.com/sindresorhus/execa/pull/246/commits/ace8ef2d6b9d82a87c6b1c7ea173d69f4a8c8db0